src/lib-components/CardMeta.vue

```diff
@@ -69,10 +76,25 @@ const props = defineProps({
     type: String,
     default: '',
   },
+  tagLabels: {
+    type: Array as PropType<ScreeningTagItemType[]>,
+    default: () => [],
+  },
+  introduction: {
+    type: String,
+    default: '',
+  },
+  isCentered: {
```

Comment:
Along with isCentered, also add useRoute to check if it is a collection home page path if the centered is only on the collection homepage to show centered content, Ask UX if not sure about the route. See getSectionName in DividerWayFinder

Reply:
Update:
the centering is only on landing pages - if a collection has items inside
So we are going to do the centering on the page level.